apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -33,7 +33,13 @@ public TransactionTests(TransactionTableFixture fixture)
 
         private async Task IncrementByOneAsync(SpannerConnection connection, bool orphanTransaction = false)
         {
-            var backoffSettings = new BackoffSettings(TimeSpan.FromMilliseconds(250), TimeSpan.FromSeconds(5), 1.5);
+            var retrySettings = RetrySettings.FromExponentialBackoff(
```

Comment:
Maybe we should consider using the RetryHelpers here as well. Maybe add a TODO?

Reply:
I'd rather keep this minimal for the moment, to be honest.